packages/renderer/src/components/IpcRender.vue

```diff
@@ -0,0 +1,19 @@
+<script lang="ts" setup>
+import { ref } from 'vue';
+const msg = ref('');
+
+function sendMsg(){
+window.ipcRenderer.send('renderMsg','ping');
+ window.ipcRenderer.on('mainMsg',(e,a)=>{
+    msg.value = a;
+  });
+}
+</script>
+<template>
+  <button
+    @click="sendMsg"
+  >
+    send
+  </button>
+  <p>ping -> {{ msg }}</p>
```

Comment:
Create a input for question text and send it to the `main` instead hardcoded `ping`.
Show index of the clicked button from dialog.